packages/amplify-e2e-core/src/utils/sdk-calls.ts

```diff
@@ -145,6 +145,39 @@ export const getUserPool = async (userpoolId, region) => {
   return res;
 };
 
+export const deleteUserPoolDomain = async (domain: string, userpoolId: string, region: string) => {
+  config.update({ region });
+  let res;
+  try {
+    res = await new CognitoIdentityServiceProvider().deleteUserPoolDomain({ Domain: domain, UserPoolId: userpoolId }).promise();
+  } catch (e) {
+    console.log(e);
+  }
+  return res;
+};
+
+export const deleteSocialIdpProviders = async (providers: string[], userpoolId: string, region: string) => {
+  config.update({ region });
+  for await (const provider of providers) {
```

Comment:
for await...of is needed for async iterables, but here providers is an array of strings

```suggestion
  for (const provider of providers) {
```

Reply:
my bad